dls-controls/malcolmjs
# Feature/auto layout #253

## Description

Added auto-layout functionality to the layout view

## Testing instructions

Add a set up instructions describing how the reviewer should test the code
- Review code
- Check Travis build
- Review changes to test coverage
- Run the site, go to the layout and click auto-layout in the top right.

## Agile board tracking

connect to #253 
connect to #256 

closes #253 
closes #256
